post-microsservice/src/main/kotlin/com/example/postmicrosservice/controller/AuthorController.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.postmicrosservice.controller
+import com.example.postmicrosservice.model.Author
+import com.example.postmicrosservice.service.AuthorService
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/author")
+class AuthorController (
+    private val authorService: AuthorService
+) {
+    @GetMapping("/{id}")
+    fun get(@PathVariable id: Long): ResponseEntity<Author> {
+        val author = authorService.getAuthor(id)
+        return if (author != null) {
+            ResponseEntity<Author>(author, HttpStatus.OK)
+        } else {
+            ResponseEntity<Author>(null, HttpStatus.NOT_FOUND)
+        }
```

Comment:
I think u can do this too:
return author?.let { ResponseEntity.ok(it) } ?: ResponseEntity.notFound().build()

Try it!